waku/v2/protocol/lightpush/waku_lightpush_option.go

```diff
@@ -58,7 +66,7 @@ func WithPeer(p peer.ID) RequestOption {
 func WithPeerAddr(pAddr multiaddr.Multiaddr) RequestOption {
 	return func(params *lightPushRequestParameters) error {
 		params.peerAddr = pAddr
```

Comment:
Should the same behavior of appending to a list of addresses be added here?

Reply:
That is not required as it gets handled here already 

https://github.com/waku-org/go-waku/blob/23b23b02b42420013b5c7356e75b194c486cc216/waku/v2/protocol/lightpush/waku_lightpush.go#L279-L286